www/src/components/ecosystem/ecosystem-board.js

```diff
@@ -0,0 +1,71 @@
+import React from "react"
+import PropTypes from "prop-types"
+import styled from "react-emotion"
+
+import EcosystemSection from "./ecosystem-section"
+
+import presets, { colors } from "../../utils/presets"
+
+const EcosystemBoardRoot = styled("div")`
+  display: flex;
+  flex-direction: column;
+
+  ${presets.Tablet} {
+    flex-direction: row;
+    height: calc(
+      100vh - (${presets.bannerHeight} + ${presets.headerHeight} + 1px)
+    );
+    padding: 2rem 1rem;
+  }
+`
+
+const EcosystemBoard = ({
+  icons: { plugins: PluginsIcon, starters: StartersIcon },
+  starters,
+  plugins,
+}) => {
+  return (
+    <EcosystemBoardRoot>
+      <EcosystemSection
+        title="Plugin Library"
+        description="With 429 plugins, lorem ipsum sunt we need some proper copy here pulling in data from your favorite source is only a few mouseclicks away."
+        subTitle="Fetured Plugins"
```

Comment:
"Featured" (minor, just noting it while we see it!)

Reply:
:+1: